package.json

```diff
@@ -4,11 +4,16 @@
   "main": "src/main/webapp/main.js",
   "license": "MIT",
   "devDependencies": {
+    "@babel/core": "^7.8.4",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.8.4",
```

Comment:
figure out how dependencies, peerDependencies and devDependencies should be structured